common/database/repositories/ClientCardRepo.ts

```diff
@@ -35,7 +35,6 @@ class ClientCardRepo {
         const qu = query.where('email', '==', email);
 
         const snapshot = await qu.get();
-
```

Comment:
Let's try to avoid unrelated whitespace changes wherever possible

Reply:
Got it! I'm not seeing that files in the list of changed files though?